exampleCourse/questions/demo/surveyQuestion/question.html

```diff
@@ -0,0 +1,48 @@
+<pl-question-panel>
+
+        <p>Please answer the following demographic survey questions. </p>
```

Comment:
Nit: we don't use 8 spaces of indentation elsewhere. Let's try to remain consistent.

Reply:
Fixed here https://github.com/PrairieLearn/PrairieLearn/pull/1867/commits/1fe5f5fe4ef56d022cb77a461a1d73e7ab80596e